src/client/app/translations/data.js

```diff
@@ -132,6 +132,7 @@ const localeData = {
 		"hide": "Hide",
 		"hide.options": "Hide options",
 		"hide.options.in.link": "Hide menu in link",
+		"here": "here",
```

Comment:
This is slightly off for where it should be alphabetically. It should be before "hide".

Reply:
I'm fixed this.